pkgs/development/python-modules/mss/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildPythonPackage, fetchPypi, isPy3k }:
+
+buildPythonPackage rec {
+  pname = "mss";
+  version = "6.0.0";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0dicp55adbqxb7hqlck95hngb1klv5s26lszw3xim30k18bwqaxl";
+  };
+
+  propagatedBuildInputs = [ ];
+
+  prePatch = ''
+    # By default it attempts to build Windows-only functionality
```

Comment:
please put comments outside of code blocks as much as possible. Helps to reduce rebuilds when editing comments.

Reply:
Done.